oh2.js

```diff
@@ -123,6 +130,77 @@ function turnOnOff(context, event) {
     rest.postItemCommand(event.payload.accessToken, event.payload.appliance.applianceId, state, success, failure);
 }
 
+function getLockState(context, event) {
```

Comment:
Can you move the lock functions below the adjustTemperatureWithItems function?  This way we keep the on/off and Percent calls, temp calls, and lock calls organized.